[Bug]: "Central Log Store is not initialized!!!" error when running GetXMLDataTest

### Apache Hop version?

2.16.0-SNAPSHOT-20251013

### Java version?

OpenJDK 64-Bit Server VM Microsoft-11369865 (build 17.0.15+6-LTS, mixed mode, sharing)

### Operating system

Windows

### What happened?

When running GetXMLDataTest (and other transform integration tests) with Maven, tests fail with:

```
[INFO] Results:
[INFO] [ERROR] Errors:
[ERROR] GetXMLDataTest.testGetXMLDataSimple1:324 » Runtime Central Log Store is not initialized!!!
[ERROR] GetXMLDataTest.testInit:426 » Runtime Central Log Store is not initialized!!!
```
This happens even though HopEnvironment.init() is called in the test. The issue does not occur when running tests directly from IntelliJ. Also, using the repo of 2 weeks ago, it runs fine even with Maven. I get the error when I run:
`mvn -T 1C -pl plugins/transforms/xml package -am`
or
`mvn -T 1C clean install`

Initializing the Hop engine environment at the test class solves the issue.

### Issue Priority

Priority: 2

### Issue Component

Component: Other

--- a/plugins/transforms/xml/src/test/java/org/apache/hop/pipeline/transforms/xml/getxmldata/GetXMLDataTest.java
+++ b/plugins/transforms/xml/src/test/java/org/apache/hop/pipeline/transforms/xml/getxmldata/GetXMLDataTest.java
@@ -23,7 +23,6 @@ import static org.junit.jupiter.api.Assertions.fail;
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.List;
-import org.apache.hop.core.HopEnvironment;
 import org.apache.hop.core.RowMetaAndData;
 import org.apache.hop.core.exception.HopValueException;
 import org.apache.hop.core.plugins.PluginRegistry;
@@ -33,6 +32,7 @@ import org.apache.hop.core.row.IValueMeta;
 import org.apache.hop.core.row.RowMeta;
 import org.apache.hop.core.row.value.ValueMetaFactory;
 import org.apache.hop.core.row.value.ValueMetaString;
+import org.apache.hop.junit.rules.RestoreHopEnvironmentExtension;
 import org.apache.hop.pipeline.Pipeline;
 import org.apache.hop.pipeline.PipelineHopMeta;
 import org.apache.hop.pipeline.PipelineMeta;
@@ -44,9 +44,12 @@ import org.apache.hop.pipeline.transforms.dummy.DummyMeta;
 import org.apache.hop.pipeline.transforms.injector.InjectorMeta;
 import org.apache.hop.pipeline.transforms.xml.RowTransformCollector;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 
 /** Test class for the "Get XML Data" transform. */
+@ExtendWith(RestoreHopEnvironmentExtension.class)
 class GetXMLDataTest {
+
   public IRowMeta createRowMetaInterface() {
     IRowMeta rm = new RowMeta();
 
@@ -192,9 +195,6 @@ class GetXMLDataTest {
    */
   @Test
   void testGetXMLDataSimple1() throws Exception {
-    HopEnvironment.init();
-
-    //
     // Create a new pipeline...
     //
 
@@ -345,10 +345,6 @@ class GetXMLDataTest {
 
   @Test
   void testInit() throws Exception {
-
-    HopEnvironment.init();
-
-    //
     // Create a new pipeline...
     //
     PipelineMeta pipelineMeta = new PipelineMeta();

